src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -273,10 +286,7 @@ def upgrade():
         nullable=True,
     )
     op.add_column(
-        "privacynoticehistory",
-        sa.Column(
-            "language", sa.String(), nullable=True
-        ),  # temporarily nullable for existing values
```

Comment:
also no longer "temporarily" nullable, it's now just nullable, we won't backfill